### DISCUSSION TITLE
fsr rcas 系数正相关

### DISCUSSION THREAD

**Discussion by hooke007** [Blinue/Magpie#127] [General]
可能修改为正相关对初识fsr的用户没啥影响，甚至觉得很合理（确实也没啥不合理）

不过我先接触别的fsr再用magpie的fsr总感觉怪怪的，出于好奇就问问开发者移植的时候的想法

**Comment by Blinue** [comment]
一是保持一致性。

二是消除锐度的限制，FSR的锐度最大时值为0，在Magpie中对应的值为1，但Magpie的锐度甚至可以大于1。

三是FSR内部会先对锐度进行运算，计算出Magpie指定的值，现在就可以跳过这一步。

**Reply by hooke007** [reply]
![Snipaste_2021-09-08_21-12-15](https://user-images.githubusercontent.com/41094733/132515812-56839674-649c-402d-a4c9-3def178ea523.jpg)
那么magpie默认的0.87是不是对应官方推荐的默认0.2 （我原以为就是简单的改为了以纵轴数字为考量）

**Comment by Blinue** [comment]
RCAS sharpness = 2 ^ (-FSR sharpness)

**Reply by hooke007** [reply]
可以，感谢解答疑惑
